KomaMRIFiles/src/Sequence/Pulseq.jl

```diff
@@ -568,8 +610,14 @@ Reads the ADC. It is used internally by [`get_block`](@ref).
 - `adc`: (`1x1 ::Vector{ADC}`) ADC struct
 """
 function read_ADC(adcLibrary, i)
+
+    if isempty(adcLibrary) || i==0
+        return [ADC(0, 0)]
+    end
+
     #Unpacking
     #(1)num (2)dwell (3)delay (4)freq (5)phase
+    # It is needed a review for this, the dwell definition may cause problems
     if !isempty(adcLibrary) # Is this the best? maybe defining i=0 is better, it works with RFs(?)
         a = adcLibrary[i]["data"]
     else
```

Comment:
This part does not make sense given the changes above.

Reply:
Addressed in [5baa0a0](https://github.com/JuliaHealth/KomaMRI.jl/pull/321/commits/5baa0a04f4da6c922c7b6851a8400d3081af278a)